token/program-2022/src/extension/transfer_fee/mod.rs

```diff
@@ -32,63 +35,71 @@ pub struct TransferFee {
     pub transfer_fee_basis_points: PodU16,
 }
 impl TransferFee {
+    /// Calculate ceiled division
```

Comment:
nit: could you provide a little more info in the comment about this implementation?